common/shared.c

```diff
@@ -496,7 +491,6 @@ int ft_finalize(
 	msg.desc = NULL;
 	msg.iov_count = 1;
 	msg.addr = addr;
-	msg.context = &tx_ctx;
```

Comment:
Also, which providers require `fi_context`? Is the fabtests suite periodically run by those providers?